### PR TITLE
Added information on Windows Autopilot Service

### PR DESCRIPTION
**Pull request by mmarable**
Trying to explain where the device registration data is stored to address concerns and pushback we are experiencing internally as the belief is that the OEM has been granted permissions within our tenant and is modifying the data directly there.

### CODE REVIEW

**Comment by PRMerger14** [comment]
@mmarable : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/autopilot/oem-registration.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ When you purchase devices from an OEM, that OEM can automatically register the d
 
 OEMs must follow [device guidelines](autopilot-device-guidelines.md) for Windows Autopilot devices.
 
+## Windows Autopilot Service
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/oem-registration.md:39]:
```suggestion
## Windows Autopilot
```

File: memdocs/autopilot/oem-registration.md [hunk 2]
```diff
@@ -36,6 +36,10 @@ When you purchase devices from an OEM, that OEM can automatically register the d
...
+
+The Windows Autopilot Service is managed and maintained by Microsoft.  This service provides the backend database that associates hardware hashes with customer tenants. When an OEM registers devices for a customer, they are writing that data to this database and not directly to the customer's tenant. No permissions to the customer's tenant are granted or required for OEMs to register devices on the customer's behalf.
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/oem-registration.md:41]:
```suggestion
Windows Autopilot is managed and maintained by Microsoft.  This service provides the backend database that associates hardware hashes with customer tenants. When an OEM registers devices for a customer, they are writing that data to this database and not directly to the customer's tenant. No permissions to the customer's tenant are granted or required for OEMs to register devices on the customer's behalf.
```

**Comment by PRMerger17** [comment]
@mmarable : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — COMMENTED** [review]
@mmarable, could you kindly also commit the first suggested change (heading)?

**Comment by PRMerger12** [comment]
@mmarable : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by PRMerger18** [comment]
@aczechowski : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by aczechowski — APPROVED** [review]
Thanks @mmarable for the edit, I made a few tweaks, but otherwise looks good.

**Comment by mmarable** [comment]
Thank you very much.
What happens next in the process?

**Pull request merged**